doc/whats_new.rst

```diff
@@ -224,11 +226,58 @@ API changes summary
      :class:`isotonic.IsotonicRegression`. By `Jonathan Arfa`_.
 
    - The old :class:`GMM` is deprecated in favor of the new
-     :class:`GaussianMixture`. The new class compute the Gaussian mixture
-     faster than before and some of computationnal problems have been solved.
+     :class:`GaussianMixture`. The new class computes the Gaussian mixture
+     faster than before and some of computational problems have been solved.
      By `Wei Xue`_ and `Thierry Guillemot`_.
 
+   - The ``grid_scores_`` attribute of :class:`model_selection.GridSearchCV`
+     and :class:`model_selection.RandomizedSearchCV` is deprecated in favor of
+     the attribute, ``results_``.
+     Ref :ref:`model_selection_changes` for more information.
+     (`#6697 <https://github.com/scikit-learn/scikit-learn/pull/4294>`_) by
+     `Raghav R V`_.
+
+
+.. _model_selection_changes:
+
+Model Selection Enhancements and API Changes
+--------------------------------------------
+
+  - **Data Independent CV splitters enabling nested cross-validation**
+
+    The new cross-validation splitters, defined in the
+    :mod:`sklearn.model_selection`, are not initialized with any
+    data-dependent parameters. Instead they expose a :func:`split` method
+    that takes in the data and yields a generator for the different splits.
+
+    This change makes it possible to do nested cross-validation with ease,
+    facilitated by :class:`model_selection.GridSearchCV` and similar
+    utilities.
+
+    Refer `#4294 <https://github.com/scikit-learn/scikit-learn/pull/4294>`_
```

Comment:
I don't think the PR needs referring to. The users don't care how the change was developed. They want to know what's good in it for them, and how to use it.